lib/cli.rb

```diff
@@ -122,7 +122,7 @@ def self_destruct
 
     desc 'version', 'Show version'
     def version
-      say(Mastodon::Version.to_s)
+      say("#{Florence::Version} (equiv: Mastodon #{Mastodon::Version})")
```

Comment:
Sometimes this option is used by other programs to check if the correct version of a program is installed, so having the version set up like this would make it a bit difficult for a program to parse, if not abort.
```suggestion
      say(Florence::Version.to_s)
```

I think it'd be better to give just the Florence version here, and talk about maybe having a separate option to give the Mastodon version, for convenience.

Reply:
The main reason why I didn't do this is because the command-line tool is still called Mastodon, so, I wanted to make it clear that it wasn't vanilla.

Reply:
Basically, someone may try and parse this string to check a Mastodon version and think that it's a really old version, which I want to avoid by making the string look totally different.

Reply:
Ah I see. In that case it *would* be preferable to have it like that.